### PR TITLE
chore: upgrade version of Design Factory

### PR DESCRIPTION
**Pull request by kpanot**
## Related issues

- :bug: Fixes :
  - https://github.com/AmadeusITGroup/otter/security/dependabot/63 
  - https://github.com/AmadeusITGroup/otter/security/dependabot/62 
  - https://github.com/AmadeusITGroup/otter/security/dependabot/61
- :rocket: Feature : #26 
- :octocat: Pull Request : #1963 #1892

### CODE REVIEW

**Comment by kpanot** [comment]
Pull Request to fix the Scss build issue is opened to DF repository

File: apps/showcase/project.json [hunk 1]
```diff
@@ -93,7 +96,7 @@
             {
               "type": "initial",
               "maximumWarning": "500kb",
-              "maximumError": "2mb"
+              "maximumError": "2.5mb"
```
Comment (fpaul-1A) [inline apps/showcase/project.json:99]: :(

File: apps/showcase/src/style/dark-theme/dark-theme.scss [hunk 2]
```diff
@@ -1,16 +1,10 @@
-@import "highlight.js/styles/a11y-dark.css";
```
Comment (fpaul-1A) [inline apps/showcase/src/style/dark-theme/dark-theme.scss:1]: should not be removed

File: apps/showcase/src/style/dark-theme/dark-theme.scss [hunk 3]
```diff
@@ -6,11 +6,6 @@
 --bs-body-color: #ffffff;
```
Comment (fpaul-1A) [inline apps/showcase/src/style/dark-theme/dark-theme.scss:6]: should be regenerated with `--df` instead of `--bs`

Comment (kpanot) [reply apps/showcase/src/style/dark-theme/dark-theme.scss:6]: ... I noticed I remove accidentally the file in the previous update (it has been recreated by the design token process) so I reseted it but I forgot to rerun the design token :'(

**Review by fpaul-1A — APPROVED** [review]

**Review by vscaiceanu-1a — APPROVED** [review]
Just by curiosity, are the changes (or a part of them) coming from a Design Factory migration script?

**Comment by kpanot** [comment]
> Just by curiosity, are the changes (or a part of them) coming from a Design Factory migration script?

Nop, I knew there were issue in their migration script so I did not run it,
I applied the changes manually.
(More over I doubt it is working in monorepo 😑)

**Review by fpaul-1A — APPROVED** [review]

File: apps/chrome-devtools/src/polyfills.ts [hunk 4]
```diff
@@ -48,6 +48,7 @@
  * Zone JS is required by default for Angular itself.
  */
 import 'zone.js'; // Included with Angular CLI.
+import '@angular/localize/init';
```
Comment (matthieu-crouzet) [inline apps/chrome-devtools/src/polyfills.ts:51]: Why do we need that ?

File: apps/showcase/project.json [hunk 5]
```diff
@@ -53,7 +53,10 @@
         "outputPath": "apps/showcase/dist",
         "index": "apps/showcase/src/index.html",
         "browser": "apps/showcase/src/main.ts",
-        "polyfills": ["zone.js"],
+        "polyfills": [
+          "zone.js",
+          "@angular/localize/init"
```
Comment (matthieu-crouzet) [inline apps/showcase/project.json:58]: same comment here

**Review by matthieu-crouzet — APPROVED** [review]

**Pull request merged**